_posts/2016-12-x-kinetic-release.markdown

```diff
@@ -0,0 +1,44 @@
+---
+author: 130s
+comments: false
+date: 2016-12-x 00:00:00+00:00
+layout: post
+slug: firstkineticrelease
+title: MoveIt! officially released into ROS Kinetic
+media_type: image
+media_link: http://wiki.ros.org/kinetic?action=AttachFile&do=get&target=kinetic.png
+description: If you're running your robot packages with ROS Kinetic, and/or if you want to try the newest MoveIt!, start at moveit.ros.org/install
```

Comment:
This is the description that will appear on the side bar of every page - I think it should mostly just be a copy of the first sentence in the actual blog post... people who are already on the website do not need a link to the install page.